src/modules/movie/dto/genreDto.ts

```diff
@@ -0,0 +1,9 @@
+export enum Genre {
+  ACTION = 'Action',
+  HORROR = 'Horror',
+  DRAMA = 'Drama',
+  COMEDY = 'Comedy',
+  FANTASY = 'Fantasy',
+  THRILLER = 'Thriller',
+  ROMANCE = 'Romance',
+}
```

Comment:
You can add here something like 'OTHER', just in case there are some movies like authobiography, cartoons, etc.